backend/globaleaks/handlers/admin/user.py

```diff
@@ -48,6 +95,7 @@ def db_create_user(session, tid, user_session, request, language):
     :param user_session: The session of the user performing the operation
     :param request: The request data
     :param language: The language of the request
+    :param wizard: Creation of first admin
```

Comment:
todo: refactor as it breaks Separation of Concerns (SoC) or Encapsulation. Bottom layers should not be aware of upper layers.